chapters/bn/chapter1/1.mdx

```diff
@@ -0,0 +1,69 @@
+# ভূমিকা
+
+## 🤗 কোর্সে স্বাগতম!
+
+<Youtube id="00GKzGyWFEs" />
+
+This course will teach you about natural language processing (NLP) using libraries from the [Hugging Face](https://huggingface.co/) ecosystem — [🤗 Transformers](https://github.com/huggingface/transformers), [🤗 Datasets](https://github.com/huggingface/datasets), [🤗 Tokenizers](https://github.com/huggingface/tokenizers), and [🤗 Accelerate](https://github.com/huggingface/accelerate) — as well as the [Hugging Face Hub](https://huggingface.co/models). It's completely free and without ads.
```

Comment:
Should we delete this section since it seems like the translated part is just below?

```suggestion
```

Reply:
YES. I'll push the changes.